utildb/update.go

```diff
@@ -594,6 +594,12 @@ func downloadFileContents(url string, startSize int64, out *bufio.Writer) (int64
 
 // extractTarGz extracts tar file contents into location of output folder
 func extractTarGz(tarGzFile, outputFolder string) error {
+	// CodeQL: Must check tarGzFile for ".." before using
+	// https://codeql.github.com/codeql-query-help/javascript/js-zipslip/
+	if strings.Contains(tarGzFile, "..") {
+		return errors.New("tar-file contains '..', disallowed as codeql vulnerabilities.")
+	}
```

Comment:
Could you add a test please?

Reply:
What do you mean by add a test?

Reply:
I mean that we should add a test which tests that if it contains `..` it creates an error.

Reply:
I don't know if we really should inherit CodeQL checks as static test since it's already being done for us with free updates to future vulnerabilities.